.travis.yml

```diff
@@ -1,8 +1,23 @@
-language: java
-
 matrix:
   include:
+    - name: Deploy MarkBind to gh pages
+      language: node
+      node_js:
+        - '8'
```

Comment:
would it be better to just use the lts version? (there was a feature that broke a while back because of nodejs `<= 7`)

Reply:
You mean not any specific version is it? And use the version provided by travis?

Reply:
yup, the latest long term support release `lts/*` https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions